helm/charts/oathkeeper/values.yaml

```diff
@@ -111,3 +111,19 @@ tolerations: []
 
 # Configure node affinity
 affinity: {}
+
+# Extra config files that will be put to the volume at "/etc/config".
+# For example, you can define multiple additional access rules files.
+extraConfigFiles: []
+  # Example:
+  # - name: access-rules.yaml
+  #   contents: |
+  #     - id: management-api
+  #       authenticators:
+  #         - handler: noop
+
+# Extra environment variables that will be passed to the container.
+extraEnv: []
```

Comment:
All config variables are loaded from `oathkeeper.config`. Do we really need to have another way of defining config? This might in fact be a problem where you define a custom env var, forget about it, update a config key in `oathkeeper.config` and are wondering the heck why it doesn't update the config while in fact the old env var is overriding the value.

Reply:
If user defines everything via config this is fine. But sometimes envs are more convenient, personally, I configure hydra purely via env vars :)

> This might in fact be a problem where you define a custom env var, forget about it, update a config key in oathkeeper.config and are wondering the heck why it doesn't update the config while in fact the old env var is overriding the value.

Yeah, this is always a choice whether to limit configuration options for the user safety or to allow flexible configuration. By the way, as I noticed the trend, charts official helm charts repository tend to provide as much flexibility as possible. 

So we have to choose our way of configuration :)

Reply:
Since `oathkeeper.config` is at the core of this chart and also seeing that it's possible to change these config values _without restarting oathkeeper_ (it now supports hot reloading) I think that we should avoid confusion and not include `extraEnv` where hot reloading does not work (changing anything requires a restart of oathkeeper). What do you think?